go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/databasemigrationservice v1.23.1
 	github.com/aws/aws-sdk-go-v2/service/dax v1.12.0
 	github.com/aws/aws-sdk-go-v2/service/directoryservice v1.16.0
-	github.com/aws/aws-sdk-go-v2/service/dlm v1.14.2
+	github.com/aws/aws-sdk-go-v2/service/dlm v1.15.6
```

Comment:
Were there any breaking changes mentioned in the CHANGELOG for this SDK version update?